packages/appeals-service-api/src/services/appeal-user.service.js

```diff
@@ -0,0 +1,23 @@
+const { AppealUserRepository } = require('../repositories/sql/appeal-user-repository');
+const ApiError = require('../errors/apiError');
+
+const appealUserRepository = new AppealUserRepository();
+
+const createUser = async (user) => {
```

Comment:
Personally I prefer `async function createUser(user)...` format - and that'd be consistent with your controllers, but not a big deal.